src/components/ScholarshipAmountField.jsx

```diff
@@ -1,28 +1,32 @@
 import React from 'react';
 import {
-  FormLabel,
-  Radio,
-  RadioGroup,
-  FormControl,
-  FormControlLabel,
+  InputLabel,
+  Grid,
+  Select,
+  MenuItem,
   FormHelperText,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
+import RemoveIcon from '@material-ui/icons/Remove';
 import PropTypes from 'prop-types';
 import AmountType from '../types/AmountType';
 import AmountTextField from './AmountTextField';
 
-const useStyles = makeStyles({
-  amountFieldStyle: {
-    display: 'flex',
-    alignItems: 'center',
+const useStyles = makeStyles((theme) => ({
+  amountSelect: {
+    minWidth: 150,
```

Comment:
From the Figma design. 
I didn't use autoWidth because I wanted it to be consistent with whatever option the user chose. But I am noticing that you can't really notice the change in width, so `autoWidth` should be fine